plugins/dfu/fu-dfu-target-stm.c

```diff
@@ -298,7 +298,7 @@ fu_dfu_target_stm_download_element1(FuDfuTarget *target,
 			}
 
 			/* if it's erasable and not yet blanked */
-			if (fu_dfu_sector_has_cap(sector, DFU_SECTOR_CAP_ERASEABLE) &&
+			if (fu_dfu_sector_has_cap(sector, DFU_SECTOR_CAP_ERASABLE) &&
```

Comment:
Is that correct?

Reply:
Well, it matches the comment on line 300...

Reply:
You're correct, erasable is correct English.